manifests/w/Wondershare/Filmora/10.5/Wondershare.Filmora.yaml

```
@@ -0,0 +1,20 @@
+PackageIdentifier: "Wondershare.Filmora"
+PackageVersion: "10.5"
+PackageLocale: "en-US"
+Publisher: "Wondershare Software Co. Ltd."
+PackageName: "Wondershare Filmora"
+License: https://www.wondershare.com/company/end-user-license-agreement.html
```

```suggestion
License: Copyright © 2021 Wondershare.
LicenseUrl: https://www.wondershare.com/company/end-user-license-agreement.html
```
My mistake. Can you please accept this?